plasmapy/particles/tests/test_particle_class.py

```diff
@@ -1081,6 +1081,21 @@ def test_custom_particle_is_category(
 ]
 
 
+@pytest.mark.parametrize(
+    "charge, mass",
+    [
+        (1 * u.C, None),
+        (1 * u.C, 1 * u.kg),
```

Comment:
I think we could also add these cases:
```suggestion
        (1 * u.C, None),      # initialize a massless CustomParticle
        (None, 1 * u.C,),     # by symmetry
        (1 * u.C, 1 * u.kg),
        (1 * u.kg, 1 * u.C),
        (None, 1 * u.kg),    # initialize a chargeless and chonky CustomParticleBoi
        (1 * u.kg, None),    # by symmetry
```